src/css/flip.css

```diff
@@ -31,6 +31,7 @@
 
 	/* Addressing the wght axis directly, so apply the
 	font-variation-settings specifically for this demo */
+	transition: font-variation-settings 0.35s ease-in;
```

Comment:
This works nicely, but some notes: 

- Works smooth in Firefox and Safari, however Chrome lags 
- Font-style can't be animated. I had a proposal I wanted to try, but I didn't want to spend too much time for only "faking" animating the "font-style" property.

Reply:
👍 No way around transitioning between two typefaces, so this is fine!

I'd suggest a shorter transition time of `100ms` to make it feel more snappy.

Chrome indeed moves slightly jerky — let's revisit that in the generic performance check PR.